### PR TITLE
Dragon skin vest update

### PR DESCRIPTION
**Pull request by glenmack**
#### Summary
Added material - "Dragon scale", lightly buffed ceramic with extra cut and bash protection
Added item-spare part - set of 100 dragon scales
Added item-armor lamellar kevlar vest
Added Recipe - dragon scale vest

```SUMMARY: Balance "dragon skin vest update"```

#### Purpose of change
Dragon scale vests are incredibly rare, and mostly useless. They mostly come damaged, can't be repaired, and are inferior in a damaged state anyway.

They still can't be repaired but they can now be disassembled. Resulting in 1 LKV (lamellar kevlar vest, basically a thin kevlar vest with a bunch of indents and stitching for scales) and up to 8 sets of 100 dragon scales. The recipe uses 8 sets, so if you fail to recover all 8, or the vest, you'll have to find another for parts.

#### Describe the solution
Makes the dragon scale vest work somewhat like an MBR vest.

#### Additional context
Saw an old thread where Kevin mentioned that only the scales need be irreparable, and that making a new one out of two or more should be feasible.

### CODE REVIEW

File: data/json/recipes/armor/torso.json [hunk 1]
```diff
@@ -532,6 +532,21 @@
...
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_TORSO",
+    "skill_used": "tailor",
+    "difficulty": 5,
+    "time": 480000,
+    "reversible": true,
+    "decomp_learn": 4,
+    "autolearn": true,
+    "components": [
```
Comment (ZhilkinSerg) [inline data/json/recipes/armor/torso.json]:
```suggestion
    "components": [ [ [ "dragonskinempty", 1 ] ], [ [ "ceramicdisks", 8 ] ] ]
```

File: data/json/recipes/armor/torso.json [hunk 2]
```diff
@@ -532,6 +532,21 @@
...
+      [ [ "dragonskinempty", 1 ] ],
```
Comment (ZhilkinSerg) [inline data/json/recipes/armor/torso.json]:
```suggestion
```

File: data/json/recipes/armor/torso.json [hunk 3]
```diff
@@ -532,6 +532,21 @@
...
+      [ [ "ceramicdisks", 8 ] ] ]
```
Comment (ZhilkinSerg) [inline data/json/recipes/armor/torso.json]:
```suggestion
```

**Comment by glenmack** [comment]
Commited linting changes, thank you @ZhilkinSerg.

**Pull request merged**